psh.proto

```diff
@@ -22,6 +22,8 @@ message HostInfoRequest {
   optional string architecture = 4;
   optional string hostname = 5;
   optional string kernel_version = 6;
+  optional string local_ipv4_addr = 7;
```

Comment:
Using fixed32 for IPv4 and bytes for IPv6 is an efficient way to represent IP addresses.